README.md

```diff
@@ -1,19 +1,19 @@
-# Ballerina Connector SDK for WSO2 Micro Integrator
+# WSO2 Micro Integrator Module Generator SDK for Ballerina
 
 ## Overview
 
-The Ballerina Connector SDK for the WSO2 Micro Integrator enables the generation of connectors that allow the WSO2 MI to run Ballerina transformations.
+The WSO2 Micro Integrator Module Generator SDK for Ballerina enables the generation of modules that allow the WSO2 MI to run Ballerina transformations.
```

Comment:
Shall we address why someone would use Ballerina inside MI? (This might be a problem for users who think of using Java mediators for easiness)

Reply:
Updated